app/src/main/java/com/example/minwoo/pingpongchat_client/LoginActivity.java

```diff
@@ -104,4 +159,34 @@ private void handleSignInResult(Task<GoogleSignInAccount> completedTask) {
             Toast.makeText(LoginActivity.this, "로그인 실패", Toast.LENGTH_SHORT).show();
```

Comment:
when you handle exception 
you should use 
"e.printStackTrace();" for debugging.